pontoon/pretranslation/tests/test_pretranslate.py

```diff
@@ -145,51 +145,152 @@ def test_get_pretranslations_fluent_empty(
     assert response == [(pretranslated_string, None, gt_user)]
 
 
+@pytest.mark.django_db
+def test_get_pretranslations_fluent_whitespace(
+    fluent_resource, google_translate_locale, tm_user
+):
+    # Various types of whitespace should be preserved
+    fluent_string = dedent(
+        """
+        whitespace =
+            { $count ->
+                [0] { "" }
+                [1] { " " }
+                *[other] { "\t" } { "\n" }
+            }
+    """
+    )
+    fluent_entity = EntityFactory(resource=fluent_resource, string=fluent_string)
+
+    expected = dedent(
+        """
+        whitespace =
+            { $count ->
+                [0] { "" }
+                [1] { " " }
+                *[other] { "\t" } { "\n" }
+            }
+    """
+    )
+
+    # Re-serialize to match whitespace
+    pretranslated_string = serializer.serialize_entry(parser.parse_entry(expected))
+
+    response = get_pretranslations(fluent_entity, google_translate_locale)
+    assert response == [(pretranslated_string, None, tm_user)]
```

Comment:
Does this check that no content is actually pretranslated here?

Reply:
Yes. Made it more obvious.